apispec/core.py

```diff
@@ -108,15 +108,7 @@ def to_dict(self):
             security_key: self._security_schemes,
         }
 
-    def schema(
-        self,
-        name,
-        properties=None,
-        enum=None,
-        description=None,
-        extra_fields=None,
-        **kwargs
-    ):
+    def schema(self, name, definition=None, **kwargs):
```

Comment:
Rather than use an OAS 2 term, "definition", what if we use a consistent parameter name for all the component methods? "component", perhaps? "obj" could also work, since OAS calls these "Objects", but that could be conflated with Python's "object".